OneSignalSDK/unittest/src/test/java/com/test/onesignal/InAppMessageIntegrationTests.java

```diff
@@ -420,7 +420,12 @@ public void testAfterLastInAppTimeIsDisplayedOncePerSession() throws Exception {
                 .pollInterval(new Duration(10, TimeUnit.MILLISECONDS))
                 .untilAsserted(() -> {
                     assertEquals(1, OneSignalPackagePrivateHelper.getInAppMessageDisplayQueue().size());
-                    assertEquals(message1.messageId, OneSignalPackagePrivateHelper.getShowingInAppMessageId());
+                    try {
+                        assertEquals(message1.messageId, OneSignalPackagePrivateHelper.getShowingInAppMessageId());
```

Comment:
Should we assert under the second Awaitility.await() too?

Reply:
Oddly enough I never the Awaitility below for the 2nd IAM every fail. However you are right in the fact I think it could so I'll add it there too.
Fixed in commit: 0749a242133fb2dfb0bb5b053fa781050c57a07c